src/pkg/bundle/create.go

```diff
@@ -86,16 +85,56 @@ func (b *Bundle) Create() error {
 		}
 	}
 
+	// for dev mode update package ref for local bundles, refs for remote bundles updated on deploy
```

Comment:
2 suggestions:
- I think you can remove "refs for remote bundles updated on deploy"
- Lift that second conditional up, something like `if config.Dev && len(b.cfg.DevDeployOpts.Ref) != 0`